src/utils/VectorFactory.ts

```diff
@@ -14,11 +14,17 @@
  * Otherwise, we just add each point to the string
  * unaltered.
  */
-export function wktPolygonToSvg(wkt: any , boundingRect?: any) {
-  let svg: any;
-  if (!wkt || wkt.wkt === '') {
+export function wktPolygonToSvg(wkt: string , boundingRect?: any) {
```

Comment:
Might as well harden this up at some point and actually define a type for boundingRect also.